politeiawww/api/v1/v1.go

```diff
@@ -61,7 +61,15 @@ const (
 
 	// ValidProposalNameRegExp is the regular expression of a valid
 	// proposal name
-	ValidProposalNameRegExp = `^[[:alnum:]\.\:\;\,\- \@\+\#]{8,}$`
+	ValidProposalNameRegExp = `^[A-z0-9\&\.\,\;\:\-\ \@\+\#]{8,80}$`
```

Comment:
There should be a `PolicyProposalNameSupportedCharacters` variable which is an array of these characters, and `ValidProposalNameRegExp` should be dynamically constructed using that variable and `PolicyMinProposalNameLength` and `PolicyMaxProposalNameLength`.

Doing it that way will allow us to maintain exactly one version of each piece of data, but it can be reused both for the regexp and sent through the policy route. In this code there are two places where you specify 8 and 80.

Reply:
Oh, yeah! I forgot about it, it must be lost on my rebases, I will fix it. Sorry about that